docs/conf/portland/2018/news/cfp-reminder.rst

```diff
@@ -0,0 +1,35 @@
+:template: 2018/na/generic.html
+
+.. post:: January 4, 2018
+   :tags: portland-2018, website, cfp, tickets
+
+
+Write the Docs Portland: CFP Reminder & Ticket Sales Open
+#########################################################
+
+Happy New Year, fellow documentarians!
+
+Stepping across the threshold into 2018, the Portland Write the Docs conference is suddenly feeling *much* closer! Our `call for proposals closes next Wednesday <http://www.writethedocs.org/conf/portland/2018/cfp/>`_, and `tickets for the conference are officially on sale <http://www.writethedocs.org/conf/portland/2018/tickets/>`_. Read on for more details on both!
+
+Submit your talk by Wednesday
+------------------------------------------------------------
+
+With the deadline for submissions coming up on **Wednesday, January 10th at midnight PST**, there's just a few days left to propose a talk for Write the Docs Portland!
+
+If you're interested or already working on pitching a talk, make sure you review the full `call for proposals <http://www.writethedocs.org/conf/portland/2018/cfp/>`_. And if you've already got an idea, but you’re having a hard time getting your proposal to come together, you can always ask for feedback on the `WTD Slack <http://slack.writethedocs.org/>`_. The community is there to help!
```

Comment:
Slack should link to http://www.writethedocs.org/slack/.

Also this should be using one-sentence linebreaks, so I can comment on the actual line.

Reply:
link fixed

your position on line breaks is noted ;P